caktus_primer/static/stylus/modals.styl

```diff
@@ -0,0 +1,30 @@
+dialog
+    position: relative
+
+    @media all and (max-width: 600px)
+        width: 90vw !important
+
+
+    button[aria-label="open"]
+        position: absolute
+        top: 0
+        right: 0
+
+    button[aria-label="close"]
+        position: absolute;
+        height: 44pt /* Per Apple's HIG we should maintain a tappable area of 44pt by 44pt */
+        width: 44pt
+        top: 0
+        right: 0
+        color: black
+        border: none
+        background: transparent
+        box-shadow: none
+        font-weight: bold
+
+
+        button[aria-label="close"]:hover
```

Comment:
I think the indentation on this is wrong? Its making a rule that applies outside of the `dialog` containers, page-wide. Let's nest it and maybe use `&:hover` so its clearly under its parent?